sources/api/netdog/src/net_config/v1.rs

```diff
@@ -64,6 +68,12 @@ impl Interfaces for NetConfigV1 {
 
         wicked_interfaces
     }
+
+    #[cfg(net_backend = "systemd-networkd")]
+    fn as_networkd_config(&self) -> Result<NetworkDConfig> {
+        let devices = Vec::from_iter(self.interfaces.iter());
```

Comment:
*hyper* nit, not gonna hold you up on this, but I think the compiler can work out what to do here and [`collect` is mentioned in the docs as being preferred](https://doc.rust-lang.org/std/iter/trait.FromIterator.html) unless you need a turbofish, but I don't think you need one here.

```suggestion
        let devices = self.interfaces.iter().collect();
```